app/models/measurement/gr_value_loader.rb

```diff
@@ -28,14 +30,10 @@ def load_historic_gr_values
     end
 
     def build_total_hash(gr_resp)
```

Comment:
@draffensperger thanks for pushing me to clean this up. I didn't really like the code when I wrote it, I needed to come back to it. How does this look?